api/v1alpha1/types_etcd.go

```diff
@@ -10,10 +10,10 @@ import (
 	corev1 "k8s.io/api/core/v1"
 	"k8s.io/apimachinery/pkg/api/resource"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
 	"k8s.io/utils/pointer"
 )
 
-// TODO Remove unused constants
```

Comment:
These constants are to be used by any user of druid who wishes to deploy `Etcd` resources programatically, specifically using the go API we provide. For instance, gardener uses a few of these constants when creating the Etcd resource which it deploys, [here](https://github.com/gardener/gardener/blob/ece32010c9d425157256a99d59ebbf4da4955a08/pkg/component/etcd/etcd/etcd.go#L198-L200).

Though these constants may never be used within druid, they are meant to be used by consumers of the druid API.

Reply:
This is the reason that we removed the comment, since we can never know which constants are used by which users, and as long we these options are supported, the constants for them must exist too.